### PR TITLE
Update Quest README

### PR DESCRIPTION
**Pull request by daniel-rearc**
Minor changes based on feedback from candidates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,18 +5,27 @@
...
 You may do all or some of the following tasks. Please read over the complete list before starting.
 
 1. If you know how to use git, start a git repository (local-only is acceptable) and commit all of your work to it.
-2. Deploy the app in any public cloud and navigate to the index page. Use Linux 64-bit x86/64 as your OS (Amazon Linux preferred in AWS, Similar Linux flavor preferred in GCP and Azure)
-3. Deploy the app in a Docker container. Use `node` as the base image. Version `node:10` or later should work.
-4. Inject an environment variable (`SECRET_WORD`) in the Docker container. The value of `SECRET_WORD` should be the secret word discovered on the index page of the application.
-5. Deploy a load balancer in front of the app.
-6. Use Infrastructure as Code (IaC) to "codify" your deployment. Terraform is ideal, but use whatever you know, e.g. CloudFormation, CDK, Deployment Manager, etc.
-7. Add TLS (https). You may use locally-generated certs.
+2. Deploy the app in any public cloud and navigate to the index page.
```
Comment (mattnewell) [inline README.md]: I still wanna change to this:
> Deploy the app in a container in any public cloud -- using whichever services you think best solve this problem -- and navigate to the index page.

**Review by mattnewell — CHANGES_REQUESTED** [review]

**Comment by mattnewell** [comment]
Some quick background on this for @rahulsen or others that may take a look at this.

We've seen a number of quests come through where the candidate has used bare EC2 instances rather than ECS or EKS or App Runner or whatever. I interviewed one of these candidate yesterday and asked "why?" His answer: he thought it was implied in this README. Trying to clean that up a bit -- still without specifying particular solution.

Daniel made a few good other changes, as well, but the above was my primary goal.

File: README.md (hunk 1 above)
Comment (daniel-rearc) [reply README.md]: The following two items address this. This line is redundant so I removed it.

File: README.md [hunk 2]
```diff
@@ -5,18 +5,26 @@
...
-4. Inject an environment variable (`SECRET_WORD`) in the Docker container. The value of `SECRET_WORD` should be the secret word discovered on the index page of the application.
-5. Deploy a load balancer in front of the app.
-6. Use Infrastructure as Code (IaC) to "codify" your deployment. Terraform is ideal, but use whatever you know, e.g. CloudFormation, CDK, Deployment Manager, etc.
-7. Add TLS (https). You may use locally-generated certs.
+3. Deploy the app in a container in any public cloud using the services you think best solve this problem.
+   - Use `node` as the base image. Version `node:10` or later should work.
+4. Navigate to the index page.
+5. Inject an environment variable (`SECRET_WORD`) in the Docker container. The value of `SECRET_WORD` should be the secret word discovered on the index page of the application.
+6. Deploy a load balancer in front of the app.
+7. Use Infrastructure as Code (IaC) to "codify" your deployment. Terraform is ideal, but use whatever you know, e.g. CloudFormation, CDK, Deployment Manager, etc.
```
Comment (mattnewell) [inline README.md]: Sorry for the iteration -- i'm thinking this through more, each time I read it.

I don't think this should be a "step." I'm just thinking about how I did my quest, and how I think most professionals we're looking for would do it. I'm already using Terraform at step 2. I'm not doing ClickOps first, then terraform later.

I think we should pull this line out from here, and put it above step 1 as general guidance.

File: README.md [hunk 3]
```diff
@@ -5,18 +5,26 @@
...
 You may do all or some of the following tasks. Please read over the complete list before starting.
 
 1. If you know how to use git, start a git repository (local-only is acceptable) and commit all of your work to it.
-2. Deploy the app in any public cloud and navigate to the index page. Use Linux 64-bit x86/64 as your OS (Amazon Linux preferred in AWS, Similar Linux flavor preferred in GCP and Azure)
-3. Deploy the app in a Docker container. Use `node` as the base image. Version `node:10` or later should work.
-4. Inject an environment variable (`SECRET_WORD`) in the Docker container. The value of `SECRET_WORD` should be the secret word discovered on the index page of the application.
-5. Deploy a load balancer in front of the app.
-6. Use Infrastructure as Code (IaC) to "codify" your deployment. Terraform is ideal, but use whatever you know, e.g. CloudFormation, CDK, Deployment Manager, etc.
-7. Add TLS (https). You may use locally-generated certs.
+3. Deploy the app in a container in any public cloud using the services you think best solve this problem.
```
Comment (mattnewell) [inline README.md]: Markdown renders it correctly anyways, but we no longer have a step 2, and it looks weird in the code.

Comment (daniel-rearc) [reply README.md]: Typical markdown just uses 1 for all numbers in the code. I was going to change it when I deleted the line, but then all the surrounding lines would have hidden the change.  I'm good with swapping it out.

File: README.md (hunk 2 above)
Comment (daniel-rearc) [reply README.md]: Troublesome

**Review by mattnewell — APPROVED** [review]

**Review by rahulsen — APPROVED** [review]

**Pull request merged**